tpchgen-cli/tests/cli_integration.rs

```diff
@@ -175,6 +176,134 @@ async fn test_write_parquet_orders() {
     }
 }
 
+#[tokio::test]
+async fn test_write_parquet_row_group_size_default() {
+    // Run the CLI command to generate parquet data with default settings
+    let output_dir = tempdir().unwrap();
+    Command::cargo_bin("tpchgen-cli")
+        .expect("Binary not found")
+        .arg("--format")
+        .arg("parquet")
+        .arg("--scale-factor")
+        .arg("1")
+        .arg("--output-dir")
+        .arg(output_dir.path())
+        .assert()
+        .success();
+
+    expect_row_group_sizes(
+        output_dir.path(),
+        vec![
+            RowGroups {
+                table: "customer",
+                row_group_bytes: vec![6524595, 6510817, 6509412, 6519647],
+            },
+            RowGroups {
+                table: "lineitem",
+                row_group_bytes: vec![
+                    7159834, 7109252, 7093240, 7123300, 7147731, 7122707, 7144719, 7101681,
```

Comment:
here you can see the row group sizes with scale factor 1 are close to the default target of 7MB